README.md

```diff
@@ -1247,6 +1250,140 @@ echo 'Employee name: '.$employee->getName(); // Employee name: John Doe
 
 **[⬆ back to top](#table-of-contents)**
 
+## Error handling
+
+### Use Exceptions Rather Than Return Codes
+
+**Bad:**
+
+```php
+class DeviceController 
+{
+    const DEV1 = "/dev/dev1";
+    private $record;
+
+    public function sendShutDown(): void
+    {
+        $handle = $this->getHandle(self::DEV1);
+
+        if ($handle != DeviceHandle::INVALID) {
```

Comment:
Same as [here](https://github.com/jupeter/clean-code-php/pull/138/files#r169024432). This condition will always be [`TRUE`](https://3v4l.org/4IDTa).